apps/dashboard/src/app/trending/components/sorting-header.client.tsx

```diff
@@ -0,0 +1,31 @@
+"use client";
+import { ChevronDown } from "lucide-react";
+import { usePathname, useRouter, useSearchParams } from "next/navigation";
+
+export type SortingHeaderProps = {
+  sortBy: string;
+  title: string;
+};
+
+export function SortingHeader(props: SortingHeaderProps) {
+  const router = useRouter();
+  const path = usePathname();
+  const searchParams = useSearchParams();
+  const range = searchParams?.get("timeRange");
+  const currentSort = searchParams?.get("sortBy") || "transactionCount";
+  const isCurrentSort = currentSort === props.sortBy;
+  return (
+    // biome-ignore lint/a11y/useKeyWithClickEvents: <explanation>
```

Comment:
can u (inside the div) adda button with variant="link" instead nad have that have the onClick? this is a valid a11y warning, screen readers won't pick this up